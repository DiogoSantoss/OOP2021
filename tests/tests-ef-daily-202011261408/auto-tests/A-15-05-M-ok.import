SUPPLIER|P1|Publicações Europa-América|Lisboa, Portugal
SUPPLIER|P2|O’Reilly|Köln, Germany
CLIENT|R2|Jorge Figueiredo|Lisboa, Portugal
CLIENT|S2|Ricardo Figueiredo|Lisboa
BOX|1C6H5OH|NORMAL|P1|2|20|100
BOX|2H2|AIR|P2|4|20|100
BOX|3O3|EXPRESS|P2|8|20|100
BOX|4CO2|PERSONAL|P2|16|20|0
CONTAINER|6M2|AIR|C4|P2|4|20|100
CONTAINER|7M5|EXPRESS|B4|P2|8|20|100
BOOK|8B1256|Os Lusíadas|Luís de Camões|1234567890|P1|58|2|5
BOOK|B1937|How to fix almost everything|Satoshi Yamada|1928374650|P1|5|2|5
