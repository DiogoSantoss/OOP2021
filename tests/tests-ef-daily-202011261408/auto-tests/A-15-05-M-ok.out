Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Consultas
1 - Produtos com Preço Abaixo de Limite
2 - Facturas Pagas por Cliente
0 - Sair
Escolha uma opção: Identificador do cliente: Facturas Pagas por Cliente: Operação inválida: O cliente '-1' não existe.
Consultas
1 - Produtos com Preço Abaixo de Limite
2 - Facturas Pagas por Cliente
0 - Sair
Escolha uma opção: Identificador do cliente: Consultas
1 - Produtos com Preço Abaixo de Limite
2 - Facturas Pagas por Cliente
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: 